diff --git a/node_modules/web-did-resolver/lib/resolver.cjs b/node_modules/web-did-resolver/lib/resolver.cjs
index 531b1e9..5aca9d7 100644
--- a/node_modules/web-did-resolver/lib/resolver.cjs
+++ b/node_modules/web-did-resolver/lib/resolver.cjs
@@ -255,7 +255,7 @@ function getResolver() {
         path = id.map(decodeURIComponent).join('/') + '/did.json';
       }
 
-      const url = `https://${path}`;
+      const url = `http://${path}`;
       const didDocumentMetadata = {};
       let didDocument = null;
 
diff --git a/node_modules/web-did-resolver/src/resolver.ts b/node_modules/web-did-resolver/src/resolver.ts
index 00bf8db..ce2216e 100644
--- a/node_modules/web-did-resolver/src/resolver.ts
+++ b/node_modules/web-did-resolver/src/resolver.ts
@@ -21,7 +21,7 @@ export function getResolver(): Record<string, DIDResolver> {
       path = id.map(decodeURIComponent).join('/') + '/did.json'
     }
 
-    const url = `https://${path}`
+    const url = `http://${path}`
 
     const didDocumentMetadata = {}
     let didDocument: DIDDocument | null = null
